src/httpserver/http_request.hpp

```diff
@@ -275,49 +275,48 @@ class http_request
         **/
         void set_arg(const char* key, const char* value, size_t size)
         {
-            this->args[key] = std::string(value,
-                                          std::min(size, content_size_limit));
+            args[key] = std::string(value, std::min(size, content_size_limit));
         }
 
         /**
          * Method used to set the content of the request
          * @param content The content to set.
         **/
-        void set_content(const std::string& content)
+        void set_content(const std::string& content_src)
         {
-            this->content = content.substr(0,content_size_limit);
+            content = content_src.substr(0,content_size_limit);
```

Comment:
as for the other setter I commented, I'd leave this one with "this" that I find more expressive in this case